### ISSUE TITLE
MdsAsync.get(uri, contract) return null when use Logbook

### ISSUE THREAD

**Issue by ff225** [petri-lipponen-movesense/mdsflutter#6]
i'm using mdsflutter: ^2.0.0-beta2 and Datalogger & Logbook data storage feature to store data on the movesense and download when i need it.
The problem is that i get a **null value** when i call: 
```dart
var data = await MdsAsync.get(
            "suunto://MDS/Logbook/$serial/byId/$id/Data", "");
print(data);
``` 
The output is: 
```
I/flutter (32072): MdsAsync.get result: 200, {"Meas":{"Acc":[{"ArrayAcc":[{"x":-0.090930506587028503,"y":-2.0579009056091309,"z":9.5668468475341797},...

I/flutter (32072): null
``` 
The output of the first line is due to the debugPrint in the get function, the second line is the output of my print.

On checking the Mds.dart file, I noticed that the error was due to **content?["Content"]**.

In the Logbook response the field 'Content' is not present so it returns null.

**Original code**
```dart
  static Future<dynamic> get(String uri, String contract) {
    final mdscompleter = Completer<dynamic>();

    MdsImpl().get(uri, contract, (data, status) {
      debugPrint("MdsAsync.get result: $status, $data");
      final content = data.isEmpty ? {} : jsonDecode(data);
      mdscompleter.complete(content?["Content"]);
    }, (data, status) {
      debugPrint("MdsAsync.get error: $status, $data");
      mdscompleter.completeError(MdsError(status, data));
    });
    return mdscompleter.future;
  }
``` 

I changed this line and everything works correctly.
**My update**
```dart
  static Future<dynamic> get(String uri, String contract) {
    final mdscompleter = Completer<dynamic>();

    MdsImpl().get(uri, contract, (data, status) {
      debugPrint("MdsAsync.get result: $status, $data");
      final content =  data.isEmpty ? {} : jsonDecode(data);
      mdscompleter.complete(content);
    }, (data, status) {
      debugPrint("MdsAsync.get error: $status, $data");
      mdscompleter.completeError(MdsError(status, data));
    });
    return mdscompleter.future;
  }
```

**Comment by petri-lipponen-movesense** [comment]
Thanks for the report. I'll look into it.

**Comment by petri-lipponen-movesense** [comment]
Fixed in 2.0.0-beta3
